.github/workflows/shared_test.yml

```
@@ -0,0 +1,41 @@
+name: Test
+
+on:
+  workflow_call:
+    inputs:
+      maven_settings_path:
+        description: Maven settings file path
+        type: string
+        default: .maven/settings.xml
+      upload_coverage_reports:
+        description: Upload MUnit reports to GitHub Actions Artifacts
+        type: boolean
+        default: false
```

```suggestion
        default: true
```

let's set this as true by defauilt @hoangmirs

---

Changed in https://github.com/nimblehq/mulesoft-actions/pull/4/commits/f5778ba10f8a57673226045241d35c53be246d57